app/src/main/java/com/kickstarter/viewmodels/DiscoveryFragmentViewModel.java

```diff
@@ -83,6 +86,9 @@
     /** Emits a boolean that determines if an editorial should be shown. */
     Observable<Editorial> shouldShowEditorial();
 
+    /** Emits a boolean that determines if an editorial for Lights On should be shown. */
+    Observable<Editorial> shouldShowLightsOn();
```

Comment:
I was wondering, why wasn't `shouldShowEditorial` used?

Reply:
yeap! I'll change that